docs/data_preface.md

```diff
@@ -63,12 +113,35 @@ This *data preface* provides additional information about the syntax of paramete
       | 10   | DE       |        | 2070-01-01 00:00:00   | 2070-12-31 23:00:00   | 1h                     | [1,2,3,…,8760]      | v1        |          |          |           |
       ```
 
+??? note "Emission factors"
+    
+    The following emission types are considered: **co2, ch4 & n2o**. <br>
+    The 100-year time horizon global warming potentials (GWP) relative to CO2 according to the fourth IPCC assessment report (AR4) are taken as suggested 
+    by the UWB at the time of the definition in 2023: `gwp_ch4 : 25`, `gwp_n2o : 298` <br>
+ 
+    Whenever possible the assumed co2 emission factors should be consistent for same commodities across sectors. <br> 
+    **Therefore, we defined a global table `global_emission_factors`**. <br><br>
+    These are defined based on the values from the National Inventory Report for the German Greenhouse Gas Inventory from 1990-2021 (NRI). <br>
+    Please, link the emission factor column of the processes to this table and the correct column via a foreign key 
+    as explained in the note below `Linking data with foreign keys`. <br>
+
+    Emission factors for ch4 and n2o are to be defined sector- or process-specific as they are more related to attributes such as firing temperature and humidity.
+    Therefore you can directly enter numbers in the respective emission column. If you still want to define ch4 values globally e.g. for similar processes, you can
+    add an extra column to the `global_emission_factors` adding <_ch4> or <_n2o> to the column names. <br>
+
+    If you need to aggregate emission factors from multiple fuels, please add a new column to the table indicating the different
+    fuel emission factors as well as their share in the aggregation based on the assumed potentials.`agg_<percentage>_<factor1>_<percentage>_<factor2>` <br>
+    e.g. agg_30_waste_wood_wood_scraps_industry_70_waste_wood_wood_scraps_commercial_institutional"
```

Comment:
This won't work, since column headers are restricted to 52 characters

Reply:
Okay true. Is the new description fine for you?